<?xml version="1.0"?>
<launch>
    <node pkg="dist_test" type="dist_test" name="dist_test" output="screen">
        <param name="target_x" value="710"/>
        <param name="target_y" value="340"/>
        <!-- <param name="ref1_x" value="640"/> -->
        <!-- <param name="ref1_y" value="420"/> -->
        <!-- <param name="ref1_dist" value="0.12"/> -->
        <!-- <param name="ref2_x" value="985"/> -->
        <!-- <param name="ref2_y" value="590"/> -->
        <!-- <param name="ref2_dist" value="0.09"/> -->
        <param name="ref1_x" value="642"/>
        <param name="ref1_y" value="465"/>
        <param name="ref1_dist" value="0.15"/>
        <param name="ref2_x" value="1160"/>
        <param name="ref2_y" value="520"/>
        <param name="ref2_dist" value="0.08"/>
        <param name="debug" value="true"/>
        <remap from="/equirectangular/image_raw" to="/test_equirectangular/image_raw"/>
    </node>
</launch>

